src/Model/Karen.hs

```diff
@@ -120,7 +120,7 @@ parse lang =
   menuParser :: Value -> Parser Menu
   menuParser = withObject "Menu Object" $ \obj ->
     Menu
-      <$> (obj .: "dishType" >>= (.: "name"))
+      <$> (obj .: "dishType" >>= maybe (pure "Unknown menu") (.: "name"))
```

Comment:
Does this mean that Kåren has changed the payload to sometimes not return an object with a "name"-field?

Reply:
This occured once, which was the morning of today, the 14th of august. Later, around ten, the payload was updated and looked as it has done before. This is a precaution if it would happen again.